musicname/zs_abandoned_metro_v3.cfg

```
@@ -0,0 +1,5 @@
+"music"
+{
+    "madlib invazion - distant land.mp3"                    "Madlib Invazion - Distant Land"
+    "nujabes - luv sic modal soul rmx instrumental.mp3"     "Nujabes - Luv Sic (Modal Soul Rmx) Instrumental"
```

I would assume `rmx` is short for remix
```suggestion
    "nujabes - luv sic modal soul rmx instrumental.mp3"     "Nujabes - Luv Sic (Modal Soul Remix) Instrumental"
```